GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -476,10 +476,8 @@ private void ResetSelectedItemsTo(bool actsAsChild)
             RequestRefresh();
         }
 
-        private void BlameSelectedFileDiff()
+        private void BlameSelectedFileDiff(int? line = null)
         {
-            int? line = DiffText.Visible ? DiffText.CurrentFileLine : null;
```

Comment:
Sure? This argument is never used.

Reply:
fixed the caller
Too many conflicting changes right now...

Reply:
> Too many conflicting changes right now...

Be patient, please. You have thought through a lot and have integrated the history and blame functionality pretty smoothly IMO. I was afraid it would become more intrusive / complex.
I don't think it will be too difficult to support the other users' expectations step by step.

> fixed the caller

partially: the call yet

Reply:
hopefully done...
I have some difficult merges to do...